Sources/DataPipeline/DataPipelineImplementation.swift

```diff
@@ -45,6 +45,13 @@ class DataPipelineImplementation: DataPipelineInstance {
 
         // subscribe to journey events emmitted from push/in-app module to send them via datapipelines
         subscribeToJourneyEvents()
+        postProfileAlreadyIdentifiedToInAppModule()
+    }
+
+    private func postProfileAlreadyIdentifiedToInAppModule() {
+        if let identifier = analytics.userId {
```

Comment:
have we thought about the case, of an `Existing user` who updated the SDK from tracking to CDP runs the App, at this time `analytics.userId` is `nil` and this method gets called. Later we run the migration and set the `analytics.user` from `profileStore` but by that time this method had already been passed.

Reply:
You're right! I have covered this case in the [latest commit](https://github.com/customerio/customerio-ios/pull/566/commits/7c998765d046036f7a5edf4d126d521198b14429)